### PR TITLE
Fixed bug #7706. Make LogHandler of kafka-fetcher-plugin can recognize namespace.

### LINKED ISSUE #7706: [Bug] KafkaLogHandler cannot recognize namespace
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

OAP server (apache/skywalking)

### What happened

The base question is why KafkaLogHandler does not inherit from AbstractKafkaHandler from the student @codeglzhang from Summer 2021. There is some common function in the AbstractKakaHandler, such as to get an actual topic name.

### What you expected to happen

The topic name includes namespace when the user has setten namespace.

### How to reproduce

none

### Anything else

_No response_

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by codeglzhang**
Fixed the issue #7706. 

### Fix <bug description or the bug issue number or bug issue link>
- [x] Add a unit test to verify that the fix works.
I add a new unit test class (org.apache.skywalking.oap.server.analyzer.agent.kafka.provider.handler.LogHandlerTest).
- [x] Explain briefly why the bug exists and how to fix it.
This bug has been described in the #7706 . To fix it, I made `LogHandler` extend `AbstractKafkaHandler` instead of `KafkaHandler` and removed `getTopic()` in the `LogHandler`.
- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #7706.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

### CODE REVIEW

File: oap-server/server-fetcher-plugin/kafka-fetcher-plugin/src/test/java/org/apache/skywalking/oap/server/analyzer/agent/kafka/provider/handler/LogHandlerTest.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+import static org.junit.Assert.assertEquals;
+
+public class LogHandlerTest {
+    private static final String TOPIC_NAME = "skywalking-logs";
+    private LogHandler handler = null;
+    private KafkaFetcherConfig config = new KafkaFetcherConfig();
+
+    private ModuleManager manager;
+
+    @Before
```
Comment (kezhenxu94) [inline oap-server/server-fetcher-plugin/kafka-fetcher-plugin/src/test/java/org/apache/skywalking/oap/server/analyzer/agent/kafka/provider/handler/LogHandlerTest.java:46]: The `setup` method can be simplified with

```java

    @Before
    public void setup() {
        final ModuleManager manager = mock(ModuleManager.class, RETURNS_DEEP_STUBS);
        when(manager.find(LogAnalyzerModule.NAME).provider().getService(any()))
            .thenReturn(mock(ILogAnalyzerService.class));
        when(manager.find(TelemetryModule.NAME).provider().getService(any()))
            .thenReturn(mock(MetricsCreator.class));
        handler = new LogHandler(manager, config);
    }
```

File: oap-server/server-fetcher-plugin/kafka-fetcher-plugin/src/test/java/org/apache/skywalking/oap/server/analyzer/agent/kafka/provider/handler/LogHandlerTest.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                    }
+                });
+            }
+        };
+        handler = new LogHandler(manager, config);
+    }
+
+    @Test
+    public void testGetTopic() {
+        Assert.assertEquals(handler.getTopic(), TOPIC_NAME);
```
Comment (kezhenxu94) [inline oap-server/server-fetcher-plugin/kafka-fetcher-plugin/src/test/java/org/apache/skywalking/oap/server/analyzer/agent/kafka/provider/handler/LogHandlerTest.java:87]:
```suggestion
        assertEquals(handler.getTopic(), TOPIC_NAME);
```

**Review by kezhenxu94 — APPROVED** [review]
Hey @codeglzhang nice to see you come back again to contribute after our collaboration on Code of Summer 2020 😄 I just left some comments, otherwise LGTM. Thanks!

File: oap-server/server-fetcher-plugin/kafka-fetcher-plugin/src/main/java/org/apache/skywalking/oap/server/analyzer/agent/kafka/provider/handler/LogHandler.java [hunk 3]
```diff
@@ -41,25 +41,26 @@
 
     public LogHandler(final ModuleManager moduleManager,
                       final KafkaFetcherConfig config) {
+        super(moduleManager, config);
         this.config = config;
         this.logAnalyzerService = moduleManager.find(LogAnalyzerModule.NAME)
-                                               .provider()
-                                               .getService(ILogAnalyzerService.class);
+                .provider()
```
Comment (kezhenxu94) [inline oap-server/server-fetcher-plugin/kafka-fetcher-plugin/src/main/java/org/apache/skywalking/oap/server/analyzer/agent/kafka/provider/handler/LogHandler.java:47]: You should import our code style settings file and these changes will not be made

File: oap-server/server-fetcher-plugin/kafka-fetcher-plugin/src/test/java/org/apache/skywalking/oap/server/analyzer/agent/kafka/provider/handler/LogHandlerTest.java (hunk 1 above)
Comment (codeglzhang) [reply oap-server/server-fetcher-plugin/kafka-fetcher-plugin/src/test/java/org/apache/skywalking/oap/server/analyzer/agent/kafka/provider/handler/LogHandlerTest.java:46]: Hi @kezhenxu94 , thanks for your comments. I agree with this suggestion. It makes the code cleaner.  I will update the code right away.

File: oap-server/server-fetcher-plugin/kafka-fetcher-plugin/src/main/java/org/apache/skywalking/oap/server/analyzer/agent/kafka/provider/handler/LogHandler.java (hunk 3 above)
Comment (codeglzhang) [reply oap-server/server-fetcher-plugin/kafka-fetcher-plugin/src/main/java/org/apache/skywalking/oap/server/analyzer/agent/kafka/provider/handler/LogHandler.java:47]: ok, this is my negligence. I import the code style and fix it immediately

**Review by kezhenxu94 — APPROVED** [review]
Thank you @codeglzhang !!

**Review by dmsolr — APPROVED** [review]
LGTM. Thank you.

**Pull request merged**